KaktusWatch/Core/Mailing.cs

```diff
@@ -44,7 +37,7 @@ public static IEnumerable<int> GetPromotionTimeFrame(string message)
             foreach (string word in message.Split(' '))
             {
                 int hour;
-                if (Int32.TryParse(word.Replace(".", ""), out hour))
```

Comment:
Looks worse IMO, but I'll change it back. On a related note, why'd you opted for IEnumerable in this case instead of a collection that has Count property?